src/app/pages/Login/Login.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import Button from '../../components/Button/Button';
+import PasswordIcon from '../../components/Icons/PasswordIcon';
+import UserIcon from '../../components/Icons/UserIcon';
+import LabeledInput from '../../components/LabeledInput/LabeledInput';
+import styles from './Login.module.css';
+
+function Login(): JSX.Element {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    alert(`Login with ${username}:${password}`);
+  }
+
+  return (
+    <div className={styles.container}>
+      <header>
+        <h1 className={styles.header__headline}>Willkommen bei Timeonation</h1>
+      </header>
+      <main>
+        <form onSubmit={handleSubmit} className={styles.form}>
+          <p className={styles.form__headline}>
+            Melde dich mit deinen Daten an
+          </p>
```

Comment:
Maybe add a headline HTML element if you have the className `headline`? 
```suggestion
          <h2 className={styles.form__headline}>
            Melde dich mit deinen Daten an
          </h2>
```

Reply:
You're right. I changed the class name to styles.form__intro